example/lib/wifi_screen/wifi_bloc.dart

```diff
@@ -48,7 +52,7 @@ class WiFiBlocSoftAP extends Bloc<WifiEvent, WifiState> {
 
 
 
-      var listWifi = await prov.startScanWiFi();
+      var listWifi = null;/* await prov.startScanWiFi() */;
```

Comment:
Disabling startScanWiFi is a big move. It's one of the major features of the library.

Reply:
Already fixed in https://github.com/omert08/esp_softap_provisioning/commit/e94575954badec04d9f3330df912de70bf16b120